extensions/amp-analytics/0.1/config.js

```diff
@@ -63,12 +65,62 @@ export class AnalyticsConfig {
   loadConfig() {
     this.win_ = this.element_.ownerDocument.defaultView;
     this.isSandbox_ = this.element_.hasAttribute('sandbox');
+    // eslint-disable-next-line no-undef
+    const fetchVendorConfigPromise = ANALYTICS_VENDOR_SPLIT
+      ? this.fetchVendorConfig_()
+      : Promise.resolve();
 
-    return this.fetchRemoteConfig_()
+    return Promise.all([this.fetchRemoteConfig_(), fetchVendorConfigPromise])
       .then(this.processConfigs_.bind(this))
       .then(() => this.config_);
   }
 
+  /**
+   * Constructs the URL where the given vendor config is located
+   * @private
+   * @param {string} vendor the vendor name
+   * @return {string} the URL to request the vendor config file from
+   */
+  getVendorUrl_(vendor) {
+    const rtv = getMode().rtvVersion;
+    const baseUrl = getMode().localDev ? `/dist` : `${urls.cdn}/rtv/${rtv}`;
+    const max = getMode().minified ? '' : '.max';
+    // bg has a special canary config
+    const canary = vendor === 'bg' && isCanary(self) ? '.canary' : '';
```

Comment:
Realize one thing. The canary will never be true for experiments. 
Because when building experiment binary, we will always use the production config which is the same as RC. 
So in theory, no need for this until the experiment is shipped.

Reply:
Should we still keep this code here for when the experiment is shipped?

Reply:
oh yes. Let's keep this as it is